apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -62,17 +77,25 @@ class LibraryCreationDialog extends React.Component {
           dashboard.project.getCurrentName()
         ),
         librarySource: response.source,
-        loadingFinished: true,
+        loadingState: LoadingState.DONE_LOADING,
         sourceFunctionList: libraryParser.getFunctions(response.source)
       });
     });
   };
 
   handleClose = () => {
-    this.setState({loadingFinished: false});
+    this.setState({loadingState: LoadingState.LOADING});
     this.props.onClose();
   };
 
+  copyChannelId = () => {
+    let channelId = document.getElementById('library-sharing');
```

Comment:
Can you use state rather than searching the dom for the element?

Reply:
This seemed a like a good enough solution. `clipboard` isn't as supported in browsers as I would like for something like this and I wasn't sure if it was worth bringing in an additional library. Happy to discuss further.

Reply:
Might be worth a quick chat - I'm curious about the clipboard :) Shouldn't block this PR though.